app/assets/stylesheets/content/_wiki.sass

```diff
@@ -28,26 +28,48 @@
 
 div.wiki
   font-size: $wiki-default-font-size
-  $wiki-max-heading-font-size: 24px
-  line-height: 1.6em
+  $wiki-max-heading-font-size: $wiki-default-font-size * 1.5
 
-  // this should be the same as a normal heading (see toolbar)
+  // this needs to be the same as a normal level 2 heading (see toolbar)
   h1
     padding: 12px 0 0 0
     border: 0
     font-size: $wiki-max-heading-font-size
-    line-height: 1.4
+    line-height: $wiki-max-heading-font-size * 1.2
     margin-bottom: 16px
 
   h2
-    font-size: $wiki-max-heading-font-size * 0.8
+    $margin-top-size: $wiki-max-heading-font-size * 1.5
+    $margin-bottom-size: $margin-top-size * 0.6
+    margin: $margin-top-size 0 $margin-bottom-size 0 !important
```

Comment:
I do not think that we should use different margins for every heading.I propose to keep the old values for all headings. An exception may be the level 1 heading. To avoid the `!important!` you should delete the obsolete code in the `misc_legacy.sass`. This can be found here: https://github.com/opf/openproject/blob/dev/app/assets/stylesheets/_misc_legacy.sass#L389